File: api/code-samples/python/klay/others/getRawTransactionByBlockHashAndIndex.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0x29b6cd965c7d9a53a6f068da259dce1d3810ba79fff8eebac5d4da14754e67e6"
index = "0x20965255"

sdk = OpenSDK(host)
klay_response = sdk.klay.get_raw_transaction_by_block_hash_and_index(blockHash, index)

print(json.loads(klay_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/eth/transaction/getRawTransactionByBlockHashAndIndex.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const blockHash =
    "0x4c4cbf242a80183d2ea2daf047c578d5fc89c0b14c4262606c8b6bb0b36715be";
  const index = "0x0";

  provider.eth
    .getRawTransactionByBlockHashAndIndex(
      blockHash,
      index,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
